src/csvcubed/utils/tableschema.py

```diff
@@ -29,13 +26,13 @@
     FailedToReadCsvwFileContentException,
     InvalidCsvwFileContentException,
 )
-from csvcubed.utils.sparql import path_to_file_uri_for_rdflib
+from csvcubed.utils.sparql_handler.sparql import path_to_file_uri_for_rdflib
 
 _logger = logging.getLogger(__name__)
 
 
 @dataclass
-class MetadataProcessor:
+class TableSchemaManager:
```

Comment:
Think we could call this something more generic like `CsvwRdfManager` ?

Reply:
Yes makes sense